web/src/app/modules/shared/components/presentation/stepper/stepper.component.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  Component,
+  OnInit,
+  Input,
+  SimpleChanges,
+  EventEmitter,
+  Output,
+  OnChanges,
+  OnDestroy,
+  isDevMode,
+} from '@angular/core';
+import {
+  StepperView,
+  View,
+  ActionField,
+  StepItem,
+} from '../../../models/content';
+import {
+  FormGroup,
+  FormBuilder,
+  Validators,
+  ValidatorFn,
+} from '@angular/forms';
+import { WebsocketService } from '../../../services/websocket/websocket.service';
+
+interface Choice {
+  label: string;
+  value: string;
+  checked: boolean;
+}
+
+@Component({
+  selector: 'app-stepper',
+  templateUrl: './stepper.component.html',
+  styleUrls: ['./stepper.component.sass'],
+})
+export class StepperComponent implements OnChanges, OnDestroy, OnInit {
+  private v: StepperView;
+
+  @Input() set view(v: View) {
+    this.v = v as StepperView;
+  }
+  get view() {
+    return this.v;
+  }
+
+  @Output()
+  submit: EventEmitter<FormGroup> = new EventEmitter(true);
+
+  @Output()
+  cancel: EventEmitter<boolean> = new EventEmitter(true);
+
+  formGroup: FormGroup;
+  action: string;
+  steps: StepItem[];
+
+  constructor(
+    private formBuilder: FormBuilder,
+    private websocketService: WebsocketService
+  ) {
+    this.steps = [] as StepItem[];
+  }
+
+  ngOnChanges(changes: SimpleChanges): void {}
```

Comment:
We should handle stepper definition changes and update the component.  Without that the `view` knob in Storybook will never work.
(if you feel the stepper definition will remain static, than maybe just remove the knob)

Reply:
I'll remove it for now, we decided the initial implementation of the stepper would be static.